### ISSUE TITLE
Bug subscribing to page store outside svelte component

### ISSUE THREAD

**Issue by hahn-kev** [sillsdev/languageforge-lexbox#229]
Seeing the following error the first time navigating to a page after the server has started
```
[ui]server-error-hook Error: Cannot subscribe to 'page' store on the server outside of a Svelte component, as it is bound to the current request via component context. This prevents state from leaking between users.For more information, see https://kit.svelte.dev/docs/state-management#avoid-shared-state-on-the-server
[ui]    at get_store (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/app/stores.js:77:9)
[ui]    at Object.subscribe (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/app/stores.js:32:24)
[ui]    at Module.subscribe (/app/node_modules/.pnpm/svelte@3.59.1/node_modules/svelte/internal/index.mjs:59:25)
[ui]    at eval (/app/src/routes/+layout.svelte:25:45)
[ui]    at Object.$$render (/app/node_modules/.pnpm/svelte@3.59.1/node_modules/svelte/internal/index.mjs:2042:22)
[ui]    at eval (/app/.svelte-kit/generated/root.svelte:65:129)
[ui]    at $$render (/app/node_modules/.pnpm/svelte@3.59.1/node_modules/svelte/internal/index.mjs:2042:22)
[ui]    at Object.render (/app/node_modules/.pnpm/svelte@3.59.1/node_modules/svelte/internal/index.mjs:2050:26)
[ui]    at Module.render_response (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/page/render.js:185:29)
[ui]    at async Module.render_page (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/page/index.js:295:10)
[ui]    at async resolve (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/respond.js:405:17)
[ui]    at async traceResponse (/app/src/lib/otel/otel.server.ts:67:20)
[ui]    at async eval (/app/src/lib/otel/otel.server.ts:58:16)
[ui]    at async Module.respond (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/respond.js:274:20)
[ui]    at async file:///app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/exports/vite/dev/index.js:505:22
```

it seems like it might be coming from the `page.subscribe` on line 12, but it's hard to say since there's no subscribe at line 25 where the logs indicate.

trace here: https://ui.honeycomb.io/sil-language-forge/environments/test/datasets/lexbox-sveltekit-server/result/Zj2BXsy69G/trace/3M5yn1h33Q5

**Comment by hahn-kev** [comment]
```
server-error-hook Error: Cannot subscribe to 'page' store on the server outside of a Svelte component, as it is bound to the current request via component context. This prevents state from leaking between users.For more information, see https://kit.svelte.dev/docs/state-management#avoid-shared-state-on-the-server
    at get_store (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/app/stores.js:77:9)
    at Object.subscribe (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/app/stores.js:32:24)
    at Module.subscribe (/app/node_modules/.pnpm/svelte@3.59.1/node_modules/svelte/internal/index.mjs:59:25)
    at eval (/app/src/routes/+layout.svelte:27:45)
    at Object.$$render (/app/node_modules/.pnpm/svelte@3.59.1/node_modules/svelte/internal/index.mjs:2042:22)
    at eval (/app/.svelte-kit/generated/root.svelte:65:129)
    at $$render (/app/node_modules/.pnpm/svelte@3.59.1/node_modules/svelte/internal/index.mjs:2042:22)
    at Object.render (/app/node_modules/.pnpm/svelte@3.59.1/node_modules/svelte/internal/index.mjs:2050:26)
    at Module.render_response (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/page/render.js:185:29)
    at async Module.render_page (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/page/index.js:295:10)
    at async resolve (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/respond.js:405:17)
    at async traceResponse (/app/src/lib/otel/otel.server.ts:67:20)
    at async eval (/app/src/lib/otel/otel.server.ts:58:16)
    at async Module.respond (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/respond.js:274:20)
    at async file:///app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/exports/vite/dev/index.js:505:22
server-error-hook Error: Cannot subscribe to 'page' store on the server outside of a Svelte component, as it is bound to the current request via component context. This prevents state from leaking between users.For more information, see https://kit.svelte.dev/docs/state-management#avoid-shared-state-on-the-server
    at get_store (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/app/stores.js:77:9)
    at Object.subscribe (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/app/stores.js:32:24)
    at Module.subscribe (/app/node_modules/.pnpm/svelte@3.59.1/node_modules/svelte/internal/index.mjs:59:25)
    at eval (/app/src/routes/+layout.svelte:27:45)
    at Object.$$render (/app/node_modules/.pnpm/svelte@3.59.1/node_modules/svelte/internal/index.mjs:2042:22)
    at eval (/app/.svelte-kit/generated/root.svelte:65:129)
    at $$render (/app/node_modules/.pnpm/svelte@3.59.1/node_modules/svelte/internal/index.mjs:2042:22)
    at Object.render (/app/node_modules/.pnpm/svelte@3.59.1/node_modules/svelte/internal/index.mjs:2050:26)
    at Module.render_response (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/page/render.js:185:29)
    at async Module.respond_with_error (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/page/respond_with_error.js:85:10)
    at async Module.render_page (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/page/index.js:314:10)
    at async resolve (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/respond.js:405:17)
    at async traceResponse (/app/src/lib/otel/otel.server.ts:67:20)
    at async eval (/app/src/lib/otel/otel.server.ts:58:16)
    at async Module.respond (/app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/runtime/server/respond.js:274:20)
    at async file:///app/node_modules/.pnpm/@sveltejs+kit@1.18.0_svelte@3.59.1_vite@4.3.8/node_modules/@sveltejs/kit/src/exports/vite/dev/index.js:505:22
```

**Comment by rmunn** [comment]
Both those traces have something in common: two health checks fired during the lifespan of the HTTP request, and the error triggered immediately after the second health check.

![error1](https://github.com/sillsdev/languageforge-lexbox/assets/90762/638bfeed-5b96-4d15-8d84-caab5462e24d)
![error2](https://github.com/sillsdev/languageforge-lexbox/assets/90762/7bf5c16e-ec04-4796-9224-1805873e6934)

I suspect that that's not a coincidence, but I haven't yet been able to figure out the details of how the second health check could possibly be causing this. I suspect something involving a function closing over an HTTP event, but how that would manifest as `page.subscribe` running late...? No idea yet.

**Comment by hahn-kev** [comment]
that health check could just be because the server just started up and we only see this issue on a fresh start of the server

**Comment by hahn-kev** [comment]
just ran into this issue again on develop branch locally

**Comment by hahn-kev** [comment]
most recent fix 5d8d816e48ade00399459fab226e5239017e522a didn't solve it.

**Comment by myieye** [comment]
I'm surprised: we have no evidence of this ever happening in staging or prod.
Also, it seems to have stopped since @rmunn's change.
So, I'm going to put this in done.

[Dev:](https://ui.honeycomb.io/sil-language-forge/environments/test/result/w29e5zxwGWg)
![image](https://github.com/sillsdev/languageforge-lexbox/assets/12587509/448731dc-1f5d-4495-a413-0b9fc416313b)

[Staging:](https://ui.honeycomb.io/sil-language-forge/environments/staging/result/Hm28Jh1oNfu)
![image](https://github.com/sillsdev/languageforge-lexbox/assets/12587509/899f4cee-54ae-4bf8-93a2-5009d24f4ce1)

[Prod:](https://ui.honeycomb.io/sil-language-forge/environments/prod/result/chP8Avz7tp4)
![image](https://github.com/sillsdev/languageforge-lexbox/assets/12587509/29655f0c-f181-4e85-9007-0375d1b854d1)
